test/extern/require/requireInTopMod.chpl

```diff
@@ -1,11 +1,8 @@
 module M {
-  //
-  // The following file intentionally contains syntax errors to make
-  // sure we don't add requirements for modules that are not 'use'd.
-  //
-  require "bar.h";
```

Comment:
Is `bar.h` used in other programs in that directory?  Should it be removed?

Reply:
requireInSubMod.chpl uses it.